tests/host_reference/gauge_force_reference.cpp

```diff
@@ -489,19 +492,24 @@ void gauge_force_reference(void *refMom, double eb3, void **sitelink, QudaPrecis
   param.t_boundary = QUDA_PERIODIC_T;
 
   auto qdp_ex = quda::createExtendedGauge((void **)sitelink, param, R);
+  //quda::TimeProfile dummy("blah");
```

Comment:
Could be deleted :)

Reply:
50987b1b96a55143adae9810e1e45b38b7f93b13